src/pages/learn/react/index.mdx

```diff
@@ -37,7 +37,7 @@ The short story with JSX is that you absolutely need to simply embrace it for wh
 
 If you feel comfortable with your JavaScript skills and have agreed to just accept JSX into your life, we've got an excellent Community Resource for you to explore to get started:
 
-<Course slug="the-beginner-s-guide-to-react" />
+<Course id={432490} />
```

Comment:
using numeric ids for this widget is unsustainable in a practical sense

they are very buried versus just grabbing them from the url

Reply:
I was following what `lib/courses` expects since that's the loader for this widget. I can make another one that filters by slug for this use case.

Reply:
on the rails backend i’d/slug are interchangeable. i didn’t use git blame to discover when it changed but know for sure when i made this it was using the slug. 

i suspect the change is because on the Sanity side the ID is safe/stable but not totally sure